src/idom/html.py

```diff
@@ -392,20 +406,29 @@ def script(*children: str, key: Key | None = None, **attributes: Any) -> VdomDic
     function that is called when the script element is removed from the tree, or when
```

Comment:
Unrelated to this PR...
We should add a disclaimer here to warn users not to use raw user inputs (or any potentially untrusted data source) within the script contents to avoid `XSS` attacks.

Reply:
Can you create an issue for this?